content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Control Topology Management Policies on a node
+reviewers:
+- lmdaly
+- nolancon
+- klueska
+- ConnorDoyle
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+An increasing number of systems leverage a combination of CPUs and hardware accelerators to support latency-critical execution and high-throughput parallel computation. These include workloads in fields such as telecommunications, scientific computing, machine learning, financial services and data analytics. Such hybrid systems comprise a high performance environment.
+
+In order to extract the best performance, optimizations related to CPU isolation and memory and device locality are required. However, in Kubernetes, these optimizations are handled by a disjoint set of components.
+
+_Topology Manager_ is a component in Kubelet that provides node level policies to enable these performance optimizations in an user abstract manner.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Topology Manager
+
+Currently, the CPU and Device Manager in Kubernetes make resource allocation decisions independently of each other.
+This can result in sub-optimal allocations on multiple-socketed systems, performance/latency sensitive applications
+will suffer due to these sub-optimal allocations.
+
+The Topology Manager is a new Kubelet component, that acts as a source of truth for other Kubelet components to make topology aligned resource decisions.
+
+The Topology Manager provides an interface for components, called *Hint Providers*, to send and recieve topology information. The default *best-effort* algorithm takes
+all possible NUMA Node combinations for an incoming container and chooses the best fit (being the narrowest NUMA node with resources available). The topology manager recieves
+the NUMA information from the *Hint Providers* as a bitmask and preforms a bitwise and to get the combined results, it then chooses the narrowest preffered result and stores it.
+The *Hint Providers* then use this when making the resource allocation decisions.
+
+### Configuration
+
+The Topology Manager is an alpha feature in Kubernetes v1.16 and will be disbabled by default behind a feature gate. 

```

Comment:
Covered by the feature-state shortcode [above](https://github.com/kubernetes/website/pull/15716/files#diff-43e0a44ed6b2437d8a5d7480fc2b58f0R13)
(if not, “disbabled” is a typo)